packages/fx-core/src/core/FxCoreImplementV3.ts

```diff
@@ -212,6 +212,43 @@ export class FxCoreV3Implement {
     }
   }
 
+  @hooks([ErrorHandlerMW, ProjectConsolidateMW, EnvLoaderMW(false)])
+  async previewAadManifest(inputs: Inputs): Promise<Result<Void, FxError>> {
```

Comment:
The methods in fxCoreImplementV3 are the exposed interface implementation from fxCore.
PreviewAadManifest is inside executeUserTask method. 
Remove the middlewares ion it and move it to the end of class which will make the method usage clear

Reply:
Sure